makexpi.sh

```diff
@@ -44,14 +44,9 @@ if [ -n "$1" ] && [ "$2" != "--no-recurse" ] && [ "$1" != "--fast" ] ; then
 
   # Now escape from the horrible mess we've made
   cd ..
-	XPI_NAME="$APP_NAME-$1"
-  # In this mad recursive situation, sometimes old buggy build scripts make
-  # the xpi as ./pkg :(
-  if ! cp $SUBDIR/pkg/$XPI_NAME pkg/ ; then
```

Comment:
I'm eliminating this case because if I updated it I would need to test it, and the case is too obscure and weird to bother testing. So instead I'm deciding that if we want to make builds from 2012 we will have to check out the correct tag by hand and build using that version of makexpi.sh.